src/index.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
```

Comment:
- [ ]The ReactDom is imported from ` 'react-dom/client';`. I'll suggest you update your import statement to `import ReactDOM from 'react-dom/client';`. :+1: